cdc/entry/schema_storage.go

```diff
@@ -74,8 +74,8 @@ func NewSchemaStorage(
 ) (SchemaStorage, error) {
 	var (
 		snap    *schema.Snapshot
-		err     error
 		version int64
+		err     error
```

Comment:
```suggestion
		err     error
```

Reply:
This cannot be removed, otherwise the `snap` become not used, report by the linter.